src/network_modelling/network.py

```diff
@@ -132,14 +142,20 @@ def loads_single_phase(self):
         self.non_flex_buses = self._matrix_non_flexible_buses()
 
         if len(self.net.asymmetric_load) > 0:
-            # Add single phase loads and sgen to originally loaded buses
-            houses = self.net.asymmetric_load
-            for i in range(len(houses)):
-                pp.create_load(self.net, houses['bus'][i], p_mw=0, q_mvar=0, name=f"load{i+1}")
-                pp.create_sgen(self.net, houses['bus'][i], p_mw=0, q_mvar=0, name=f"sgen{i+1}")
             # Remove asymmetric loads on the three phases
-            for i in range(len(self.net.asymmetric_load)):
-                self.net.asymmetric_load['in_service'] = False
+            self.net.asymmetric_load['in_service'] = False
```

Comment:
what is `in_service`?

Reply:
It's an attribute in pandapower that makes sure an element may exist in a network but is not used. I had run a few tests a while back and if I remember correctly, setting the asymmetric loads as off-service is the same as removing them.